a first draft for a blog post (#896)
deltachat/deltachat-pages: _posts/2024-05-31-instant-onboarding.md

you can only send out end-to-end encrypted messages. 
Here is how you can setup a secure chat: 

- add a new contact through a QR code scan, or 

- share your own contact info as an "invite link" 
  that any receiver who has installed Delta Chat 
  can tap to start a chat with you, or 

- share a "group chat invite" 
  that any receiver who has installed Delta Chat 
  can tap to join the group, or 

- get added to a secure group chat 
  that will allow you to message any group member. 

These ways of starting a chat result in "guaranteed end-to-end encryption" 
which is secure against compromised servers or networks. 
Therefore, even if you are using a classic e-mail server we recommend to use these methods. 
See the [2024 security analysis from ETH Zuerich researchers](https://delta.chat/en/2024-03-25-crypto-analysis-securejoin) for more info. 

Secure chat setup also works in mesh networks in the Amazon Rain Forest 
as well as in regions that are cut off from the wider Internet
and where users can only access a local e-mail or chatmail server. 
No central key server or global infrastructure is required. 

## New: Introduce a contact by attaching a "VCard"

> XXX add right-aligned screenshot of an attached vcard for "Mad Hatter" 

Reviewer: since the image of the attached contact can be an small cut (not full screenshot) maybe also include a small cut of the "attachment selector" area where the "attach contact" button is for better context on how to attach a contact

<img src="../assets/blog/2024-05-create-profile.jpg" style="width:200px; float:right; clear:both; margin-left:.5em; margin-bottom:.2em;" />

Nobody wants or needs more accounts these days
so let us tell you about "Chat profiles". 
When initially onboarding with Delta Chat you'll create a "Chat profile"
which technically consists of your chosen name and avatar,
and an encryption-setup, e-mail address and password. 
The new 1.46 apps by default automate the last three choices 
and only ask for your name and avatar which is shown to other users. 
You can alternatively tap "use other server" and "manual login" 
if you want to specify an existing e-mail address for your chat profile. 

Whether using a chatmail or a classic e-mail server, 
you can always replicate your chat profile to a second device,
or export it to a file and import it from any other device. 
Chat profiles are "in your hand" and do not "live" on any central server,
also not in encrypted or PIN-protected form. 

## Are we turning our back on classic e-mail? 

<img src="../assets/logos/chatmail.png" style="width:200px; float:right; clear:both; margin-left:.5em; margin-bottom:.2em;" />

No. 

> Firstly, chatmail is fully interoperable with other e-mail servers, 

Reviewer: by reading this it might sounds as it is interoperable but it is not an email server, so maybe even better to say:

```suggestion
Firstly, chatmail is a fully interoperable e-mail server, 
```

(but this is a soft recommendation, it is also good enough as-is)

and any receiver can tap it to start chatting with the contact. 
The contact attachment has the [vcard format](https://www.rfc-editor.org/rfc/rfc6350)
and contains an encryption number that is used to establish end-to-end encrypted messaging. 

## New: Instant privacy-preserving push notifications for Android 

XXX add right-aligned screenshot of a top-level notification from "Alice" or so 

On Android, "instant push notifications" are enabled if possible,
while denying Google and Apple any access to user data or metadata. 
Even the physical seizure of a chatmail server or the seizure of our "central notification proxy" 
would not reveal which e-mail address is used on a phone, 
or which Google/Apple ID is associated with a chatmail address. 
No message data and no metadata would be revealed either, also not in encrypted form. 
Check out more details in our new [push notifications FAQ section](/help#instant-delivery).

## New: "Accounts" are dead. Long live "Chat profiles"!

<img src="../assets/blog/2024-05-create-profile.jpg" style="width:200px; float:right; clear:both; margin-left:.5em; margin-bottom:.2em;" />

Nobody wants or needs more accounts these days

> so let us tell you about "Chat profiles". 

Reviewer 2: Hm... I'd be careful with making such a statement. If nobody needs more accounts, why are we creating one in the background? We just rename most of the account functionality to "profile", but we can't say that no accounts are involved... this just raises the wrong hopes and questions.

All Delta Chat apps offer multi-profile support 
which means you can have a profile for instant chat messaging 
*and* another profile with a pre-existing e-mail address for classic e-mail purposes. 

## Are we de-emphasizing e-mail addresses in the user interface? 

Yes. 

We aim to make Delta Chat more approachable to the many 
who do not care about or are even skeptical of "e-mail". 
In all likelyness that's not you 
because you wouldn't be reading this post here, then :) 
But you probably know people who might appreciate that Delta Chat 
does not require a phone number or other personal data, 
offers privacy, censorship resistance, a free "choice of server" 
as well as a convenient and reliable user experience 
that integrates games and collab tools, 
along with a growing chat bot ecosystem? 

## Chatmail makes e-mail not only fast but cheap again 

With our new chatmail-based instant onboarding system, 
e-mail addresses are becoming, once again, cheap and virtually free. 
But this time around, there is no company posturing to "do no evil" 
luring everyone to their central "ethical" service and then drop the pretense soon after. 

> Running a chatmail server is a cheap activity that we want people

Reviewer 2: ```suggestion
[Running a chatmail server](https://delta.chat/en/2023-12-13-chatmail) is a cheap activity that we want people
```
Let's link to the blog post :)

---
title: Instant onboarding and instant message delivery 
author: holga,r10s
image: ../assets/blog/secure-decent-delta.png
---

<img src="../assets/blog/2024-05-create-profile.jpg" style="width:200px; float:right; clear:both; margin-left:.5em; margin-bottom:.2em;" />

Big milestone in our 6+ years long project history: 
With the rolling 1.46 app releases you can start chatting without a pre-existing e-mail address. 
Just provide a name and then tap "Agree and continue" 
to create a chat profile using the [default chatmail server](https://nine.testrun.org/privacy.html) 
or visit other [chatmail servers](/chatmail) and tap the invite on their home page. 

Great. So creating a profile takes now only a few seconds but what then? 

## Maybe try out chatting with some bots? 

Once Delta Chat is installed, 

> you can tap [this "Public Bots" link](https://i.delta.chat/#9AF055DB87EC48A1C009B6CA55E3712A6F7D346F&a=botsindex%40nine.testrun.org&n=Public%20Bots&i=QpBSronexvP&s=nAfQ0q_JomN) 

Reviewer 3: I tried the "open chat" button/link on desktop while I had delta chat open, but nothing happened. opening the same link on android did the trick, but others might not know to try that.

there's a bit of a risk that readers might interpret "Bots" as referring to "LLM chat bots", which are becoming quite unpopular with certain crowds. it might be worthwhile to clarify that these are interactive services accessed through a chat interface?

Reviewer: actually there are also LLM chat bots among them

  that any receiver who has installed Delta Chat 
  can tap to start a chat with you, or 

- sharing a "group chat invite" 
  that any receiver who has installed Delta Chat 
  can tap to join the group, or

- being added to a secure group chat 
  which will allow you to message any group member. 

<img src="../assets/blog/2023-11-qr-scan.jpg" style="width:160px; clear:both; margin-left:.5em; margin-bottom:.2em;" />
<img src="../assets/blog/green-checkmark.svg" width="140" style="margin-left:1em;" /> 

Note that these secure chat setups work just fine in mesh networks in the Amazon Rain Forest 
or in regions that are cut off from the wider Internet
because no central key server or other global infrastructure is required. 

## New: Introduce a contact to chat partners 

<img src="../assets/blog/2024-05-vcard2.png" style="width:250px; float:right; clear:both; margin-left:.5em; margin-bottom:.2em;" />

With the 1.46 releases you can now "attach" a contact to a chat message, 
and any receiver can tap it to start chatting with the attached contact. 
The contact attachment has the [vcard format](https://www.rfc-editor.org/rfc/rfc6350)

> and contains an encryption number that is used to establish end-to-end encrypted messaging. 

Reviewer 3: maybe the average reader will continue past the phrase "encryption number" without thinking too much about it, but as a crypto-nerd it sounds a bit odd to me. Maybe this could link to a page describing the details of the what is encoded and what it's used for?

without giving Google and Apple any access to user data. 
Check out [our new push notifications FAQ section](https://delta.chat/help#instant-delivery) for more details. 

## Chat profiles: on your device and in your hand 

<img src="../assets/blog/2024-05-create-profile.jpg" style="width:200px; float:right; clear:both; margin-left:.5em; margin-bottom:.2em;" />

Delta Chat does not store or keep data at e-mail servers 
but only uses them for ephemeral message transport. 
When first onboarding with Delta Chat you'll create a "Chat profile"
which initially consists of your chosen name and avatar,
and an automatically generated encryption-setup, e-mail address and password. 
If you tap "use other server" and "manual login" 
you can manually specify an existing e-mail address and password. 

Over time, your chat profile accumulates 
contacts, chat groups, messages and media files. 
You can transfer or duplicate your chat profile to another device,
either by a QR code scan or by exporting it to a file 
and then importing and using it on any other device. 
Chat profiles are literally "in your hand" and 
are not stored on any central server,

> also not in encrypted or PIN-protected form. 

Reviewer 3: > also not in encrypted or PIN-protected form.

this gives me the impression that profiles aren't encrypted, which I suspect is not the intent.

Was this supposed to imply that it's not just an encrypted blob hosted on some server?

Lastly, while Delta Chat aims to offer easier onboarding than Whatsapp, Signal or Telegram 
you can also use and regard it as an e-mail app. 
All Delta Chat apps offer multi-profile support 
which means you can have a profile for instant chat messaging 
*and* another profile with a pre-existing e-mail address for classic e-mail purposes. 

## Are we de-emphasizing e-mail addresses in the user interface? 

Yes. 

We aim to make Delta Chat more approachable for the many folks 
who do not care about or are even critical of "e-mail". 
With some likelyness that's not you because you are reading this post here :) 
But you probably know people who don't rejoice when hearing "e-mail"
but who might appreciate that Delta Chat 

- provides a convenient and reliable messaging experience,

- does not require a phone number or any other personal data, 

- offers privacy, censorship resistance and free choice of server,

- offers in-chat games like 2048, chess or hextris,
  and in-chat tools like an editor, a calendar and a checklist. 

In light of all these features, we prefer to not require 
would-be-users first to understand what "e-mail" has to do with anything. 

> As a 20-year old women once fed back "E-mail? Isn't e-mail just there for spam and work?"

Reviewer 3: > As a 20-year old women

typo: should be "woman"

E-mail in the form of chatmail is so much more than that 
but we still think it's good to let 
e-mail addresses recede into the background of Delta Chat 
just like phone numbers typically recede into the background of classic phone-based messengers. 

## Chatmail makes e-mail cheap again 

With our new chatmail-based instant onboarding system, 
e-mail addresses are becoming, like in the early 2000s, cheap and virtually free. 
But this time around, there is no company posturing to "do no evil" 
luring everyone to their central "ethical" service and then drop the pretense soon after. 
[Running a chatmail server](https://delta.chat/en/2023-12-13-chatmail) is a cheap activity 
that we want people to be able to do on the side 
and on low-end hardware all across the world. 
Chatmail is best described as an ephemeral end-to-end encrypted 
messaging routing system running at Internet-scale. 

## Internet standards FTW or: e-mail strikes back!

How often have you heart "E-mail is dead" or 
"here is this brand new thing to replace e-mail" 
from marketing pushes throughout the last two decades? 

> How often have you heart "The Web is dead and replaced by mobile apps"

Reviewer 3: > How often have you heart

should be _heard_

only to find most mobile apps being a thinly veiled web view, anyway? 

By contrast, Delta Chat fully embraces both E-mail and Web standards 
to avoid the pitfalls and failures of past and present 
"inventing a new standard that will replace e-mail (or the Web)" efforts.

> Our new "Instant onboarding" releases cultivate and grow chatmail servers,

Reviewer 3: > Our new "Instant onboarding" releases cultivate and grow chatmail servers,

I find this sentence somewhat difficult to parse due to "releases" being both a noun and a verb. It might be worth restructuring

Or visit other [chatmail servers](../chatmail) and tap the invite on their home page. 

Great, a new chat profile in a few seconds, but what now? 

Getting in contact with others is indeed a fundamental problem for any non-mainstream messenger.
In the next two sections we suggest how to get in contact with some bots and humans,
before highlighting more news about the 1.46 release series,
and providing some of our thinking around the UX and UI direction of Delta Chat
and chatmail servers. 

## Getting in contact with chat bots and "DeltaFans" 

<img src="../assets/blog/2024-05-public-bot-avatar.png" width="100px" style="float: right; margin-left:.5em; margin-bottom:.2em;" />

Once Delta Chat is installed, you may tap invite links like the following ones: 

- To verify things work, send a message to 
  [echo@nine.testrun.org bot](https://i.delta.chat/#66B77A4C5F57A670D6E86C0767F00812105D9374&a=echo%40nine.testrun.org&n=&i=m6rDyxHRShX&s=Ta7TL-JufhZ) and observe the echo reply arriving quickly. 

- Checkout [a "Public Bots" index bot](https://i.delta.chat/#9AF055DB87EC48A1C009B6CA55E3712A6F7D346F&a=botsindex%40nine.testrun.org&n=Public%20Bots&i=QpBSronexvP&s=nAfQ0q_JomN) 

>   to receive a little web app to view and start interacting with a selection of yet more bots. 

Reviewer: maybe "in-chat app" instead of web app to make clear this is not some old school web page in browser
